packages/bridge-ui/src/domain/chain.ts

```diff
@@ -137,3 +137,8 @@ export const taiko: WagmiChain = {
   //   address: "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e",
   // },
 };
+
+// Will help us to map from chain id to RPC provider
+export const providers = new Map<number, ethers.providers.JsonRpcProvider>();
```

Comment:
I am not an expert in domain driven design but I think `providers` should be a in a domain of it's own.

Reply:
@cyberhorsey what do you think?

Reply:
Let's move it into a separate domain. I wasn't sure where to add this map, which I see as a service, but I think I get the idea of having these domains... I'll make the change

Reply:
Yep agreed, its own providers file is great!